src/NoFrixion.MoneyMoov/Api/MerchantClient.cs

```diff
@@ -146,4 +148,24 @@ public Task<MoneyMoovApiResponse<IEnumerable<PaymentAccount>>> GetAccountsAsync(
             _ => Task.FromResult(new MoneyMoovApiResponse<IEnumerable<PaymentAccount>>(HttpStatusCode.PreconditionFailed, new Uri(url), prob))
         };
     }
+
+    /// <summary>
+    /// Calls the MoneyMoov UserInvites Merchant endpoint to get the list of invites that have been
+    /// sent inviting new users to join the merchant.
+    /// </summary>
+    /// <param name="auserAccessToken">A User scoped JWT access token.</param>

```

Comment:
There's a typo in the param name, should be `userAccessToken`

Reply:
Thx, will fix.